src/styles/styles.js

```diff
@@ -3888,10 +3899,47 @@ const styles = {
         fontSize: variables.fontSizeNormal,
         marginRight: 4,
     },
+
+    timePickerInput: {
+        fontSize: 69,
+        minWidth: 56,
+    },
+    timePickerWidth100: {
```

Comment:
I think this and the few other similar ones can be replaced with `styles.w100` (from [here](https://github.com/Expensify/App/blob/36f20c9717e7bd092a24da14d3da559babfe5ee0/src/styles/utilities/sizing.ts)).

Reply:
`styles.w100` is about percentage value. I'm not sure we need to introduce such a number constant. It's not commonly spread in the code. At least I haven't seen place where I can reuse it

Reply:
Ah crap sorry, I must have misread! :sweat_smile: